modules/exploits/android/local/binder_uaf.rb

```diff
@@ -42,6 +42,13 @@ def initialize(info={})
         ],
         'DisclosureDate' => '2019-09-26',
         'SessionTypes'   => [ 'meterpreter' ],
+        'Compat' => {
+          'Meterpreter' => {
```

Comment:
TODO: Confirm if we need to generate this only if `sessiontypes` has `meterpreter` in it